tests/integration/backend_test.py

```diff
@@ -376,14 +376,14 @@ def test_classical(
     c.add_c_setreg(23, a)
     c.add_c_copyreg(a, b)
 
-    c.add_classicalexpbox_register(a + b, d)
-    c.add_classicalexpbox_register(a - b, d)
+    c.add_classicalexpbox_register(a + b, d)  # type: ignore
+    c.add_classicalexpbox_register(a - b, d)  # type: ignore
 
     if language == Language.QASM:  # remove this when division supported in QIR
-        c.add_classicalexpbox_register(a * b // d, d)
+        c.add_classicalexpbox_register(a * b // d, d)  # type: ignore
 
-    c.add_classicalexpbox_register(a << 1, a)
-    c.add_classicalexpbox_register(a >> 1, b)
+    c.add_classicalexpbox_register(a << 1, a)  # type: ignore
+    c.add_classicalexpbox_register(a >> 1, b)  # type: ignore
```

Comment:
```suggestion
    c.add_classicalexpbox_register(a << 1, a.to_list())
    c.add_classicalexpbox_register(a >> 1, b.to_list())
```

Reply:
I was not sure if this might change the way this circuit is now handled. It might be possible that the left side has less or more bits than the register we are writing to?